asQ/allatonce/form.py

```diff
@@ -59,6 +61,11 @@ def __init__(self,
 
         self.form = self._construct_form()
 
+    def time_update(self):
```

Comment:
Thanks for adding this. I think it keeps the code a bit cleaner.
Could this method have an optional argument for setting the new value for `t0` please?  It makes sense for the default behaviour to be what you have here - just moving everything on by Nt*dt - but sometimes it might be useful to be able to specify t0 explicitly.

Reply:
Resolved by 26c19c00df8ad0d134b1464ad053e42c087fd062